.env

```
@@ -2,3 +2,6 @@
 
 # Microsoft Insights Key
 APPINSIGHTS_INSTRUMENTATIONKEY=SomeRandomStringForLocalDocker
+
+DB_USERNAME=ccd
+DB_PASSWORD=ccd
```

Guessing the reason these may have been set to ccd was because of ccd-docker.. if that's the case then may need to rename env vars?
```suggestion
DB_USERNAME=hmc
DB_PASSWORD=hmc
```